src/hotspot/cpu/s390/assembler_s390.hpp

```diff
@@ -189,11 +189,6 @@ class Address {
     _index(noreg),
     _disp(0) {}
 
-  Address(Register base, Register index, intptr_t disp = 0) :
-    _base(base),
-    _index(index),
-    _disp(disp) {}
-
```

Comment:
I can live with the removal, but I guess it may be useful at some point of time. s390 supports specifying both, index and disp (unlike PPC64).

Reply:
This constructor was causing ambiguity with this one :
```
  Address(Register base, RegisterOrConstant roc, intptr_t disp = 0) :
    _base(base),
    _index(noreg),
    _disp(disp) {
    if (roc.is_constant()) _disp += roc.as_constant(); else _index = roc.as_register();
  }
```

that's why I removed it. But yeah I can try to find another workaround, any suggestion from your side ?

Reply:
Ah, got it. I'm fine with the removal. The other constructor can be used as replacement.